src/components/Modals/FiatRamps/views/RampsList.tsx

```diff
@@ -23,50 +26,68 @@ export const RampsList = () => {
         <Card.Body>
           <Text lineHeight={1.2} color='gray.500' translation='fiatRamps.titleMessage' />
           <Stack spacing={2} mt={2} mx={-4}>
-            <Button
-              width='full'
-              height='auto'
-              justifyContent='space-between'
-              variant='ghost'
-              fontWeight='normal'
-              py={2}
-              onClick={() => history.push(FiatRampsRoutes.Gem)}
-              rightIcon={<ChevronRightIcon boxSize={6} />}
-            >
-              <Flex flex={1} flexDirection='row' justifyContent='space-between' alignItems='center'>
-                <Flex flexDirection='row' justifyContent='center' alignItems='center'>
-                  <AssetIcon src={gemlogo} />
-                  <Box textAlign='left' ml={2}>
-                    <Text fontWeight='bold' translation='fiatRamps.gem' />
-                    <Text translation='fiatRamps.gemMessage' />
-                  </Box>
-                </Flex>
-                <Box>
-                  <Tag colorScheme='green' mr={2}>
-                    <Text translation='fiatRamps.buy' style={{ textTransform: 'uppercase' }} />
-                  </Tag>
-                  <Tag colorScheme='gray'>
-                    <Text translation='fiatRamps.sell' style={{ textTransform: 'uppercase' }} />
-                  </Tag>
-                </Box>
-              </Flex>
-            </Button>
-            <Button
-              width='full'
-              height='auto'
-              justifyContent='flex-start'
-              variant='ghost'
-              fontWeight='normal'
-              py={2}
-            >
-              <Flex flexDirection='row' justifyContent='center' alignItems='center'>
-                <AssetIcon src={onjunologo} />
-                <Box textAlign='left' ml={2}>
-                  <Text fontWeight='bold' translation='fiatRamps.onJuno' />
-                  <Text translation='fiatRamps.comingSoon' />
-                </Box>
-              </Flex>
-            </Button>
+            {Object.entries(supportedFiatRamps)
+              .filter(([, { readyToUse }]) => readyToUse)
+              .map(([key, fiatRampConfig]) => (
+                <Button
+                  key={key}
+                  width='full'
+                  height='auto'
+                  justifyContent='space-between'
+                  variant='ghost'
+                  fontWeight='normal'
+                  py={2}
+                  onClick={() => {
+                    setFiatRampProvider(key as FiatRamps)
```

Comment:
```typescript
export const supportedFiatRamps: Record<FiatRamps, SupportedFiatRampConfig> = {
```

Using `Object.entries()` there should give us the correct typings

Reply:
Are you sure? I got an error when I remove that cast saying `string` is not assignable to FiatRamp

Reply:
Ah yeah, every few months I stumble upon this and after a quick search, realize that because of the open-endendness nature of TS, `Object.keys()` will never be as exhaustive as we'd expect, because it uses structural subtyping.
Assertion is totally fine then~